include/ignition/utils.hh

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gz/utils.hh>
```

Comment:
Is this meant to be `gz/utils/utils.hh`?

Reply:
in general, it should only have one `utils`, so `gz/utils.hh` is correct (look at the installed paths of any `ign-*` package to confirm)

the one exception is libsdformat, which has `sdf/sdf.hh` but that one is a bit different anyway